### PR TITLE
File.readlines supports `:chomp` option

### LINKED ISSUE #453: `File.readlines(path, chomp: true)` isn't supported (standard)
Given the following code:

```ruby
File.readlines(path, chomp: true)
```

When invoked in a fake filesystem, I receive this error:

```sh
ArgumentError: wrong number of arguments (given 2, expected 1)
```

As far as I can tell, Ruby's IO class has [supported the `:chomp` option since v2.5](https://ruby-doc.org/core-2.5.0/IO.html#method-c-readlines) (although [this blog post](https://blog.bigbinary.com/2017/03/07/io-readlines-now-accepts-chomp-flag-as-an-argument.html) suggests since v2.4). The general idea is that `#chomp` will be invoked on each item returned in the array.

Are y'all willing to support this? If so, I'm happy to open a PR for it within a day or so!

### PR DESCRIPTION
**Pull request by cbortz**
Fixes #453

### CODE REVIEW

File: lib/fakefs/file.rb [hunk 1]
```diff
@@ -197,11 +197,11 @@ def self.read(path, *args)
       file.read(length)
     end
 
-    def self.readlines(path)
+    def self.readlines(path, options = {})
```
Comment (grosser) [inline lib/fakefs/file.rb:200]: kw args for better error messages ?
```suggestion
    def self.readlines(path, chomp: false)
```

**Review by grosser — APPROVED** [review]

File: lib/fakefs/file.rb (hunk 1 above)
Comment (cbortz) [reply lib/fakefs/file.rb]: @grosser Done in 7fd1d6e! Thanks for the suggestion 😁

**Comment by grosser** [comment]
thx! ... just waiting for travis 😭

**Comment by grosser** [comment]
plz rebase to fix CI

**Comment by cbortz** [comment]
Done!

**Comment by grosser** [comment]
1.3.0

**Pull request merged**